ui/app/templates/components/mfa-setup-step-one.hbs

```diff
@@ -0,0 +1,39 @@
+<p>
+  TOTP Multi-factor authentication (MFA) can be enabled here if it is required by your administrator. This will ensure that
+  you are not prevented from logging into Vault in the future, once MFA is fully enforced.
+</p>
+<form id="mfa-setup-step-one" {{on "submit" this.verifyUUID}}>
+  <MessageError @errorMessage={{this.error}} class="has-top-margin-s" />
+  <div class="field has-top-margin-l">
+    <label class="is-label">
+      Method ID
+    </label>
+
+    {{! template-lint-disable no-autofocus-attribute}}
```

Comment:
thoughts on removing the autofocus attr on the Input?

Reply:
I'm not sure if we have a consistent pattern with autofocusing but I don't have a problem with removing it.